### DISCUSSION TITLE
Get-PnPSensitivityLabel Available in the current Nightly Release only.

### DISCUSSION THREAD

**Discussion by Ofer-Gal** [pnp/powershell#2938] [Q&A]
I want to run this command once So I did the following:
`Install-Module PnP.PowerShell -AllowPrerelease -RequiredVersion 2.0.50-nightly -Force -SkipPublisherCheck
`
then in the script
`Import-Module PnP.PowerShell
$appConn = Connect-PnPOnline  -Url $appURL -ReturnConnection -ClientId $ClientId -Tenant $tenant -CertificatePath $certPath -CertificatePassword $certPass
Get-PnPSensitivityLabel -Connection $appConn
`
and I get error: **The term 'Get-PnPSensitivityLabel' is not recognized as a name of a cmdlet**

Is there a way to force the PnP.PowerShell to be Version 2.0.50-nightly ?

**Comment by poke666** [comment]
even after forcing the nightly version this cmdlet is still unavailable
